scripts/apps/authoring-react/authoring-angular-integration.tsx

```diff
@@ -144,11 +144,46 @@ function getInlineToolbarActions(options: IExposedFromAuthoring<IArticle>): IAut
         availableOffline: true,
     });
 
+    const getReadOnlyAndArchivedFrom = (): Array<ITopBarWidget<IArticle>> => {
+        const actions: Array<ITopBarWidget<IArticle>> = [];
+
+        if (item._type === 'archived') {
+            actions.push({
+                group: 'start',
+                availableOffline: false,
+                component: () => (
+                    <span>
```

Comment:
it is possible for this string to get very long. Ellipsis should appear in case it no longer fits in the toolbar - but not sooner.  It should work in `DeskAndStage` component - check it out how it works there.

Reply:
Yes, it does work just fine for that so we can reuse it there.